spec/pluck_to_tstruct_spec.rb

```diff
@@ -76,4 +128,26 @@ def eql?(other)
       }.to raise_error(SorbetRails::PluckToTStruct::UnexpectedType)
     end
   end
+
+  context "given associations mappings that don't exist in T::Struct" do
+    it 'should raise error' do
+      expect {
+        plucked = Wizard.pluck_to_tstruct(TA[WizardWithWandT].new, associations: { wood_type: "wands.wood_type", broken: "wands.broken" })
```

Comment:
do you need to assign a var here?

Reply:
We don't but on line 127 we do so I figured I'd just follow what the existing test is doing

Reply:
i suspect rubocop would pick this up as a useless variable

Reply:
nit: can you reformat this code
```suggestion
        Wizard.pluck_to_tstruct(
          TA[WizardWithWandT].new, 
          associations: { wood_type: "wands.wood_type", broken: "wands.broken" }
        )
```

> We don't but on line 127 we do so I figured I'd just follow what the existing test is doing

^^ My bad, please feel free to drop the unused variable.